CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @NordSecurity/libtelio-team
+.github/workflows @Jauler @jjanowsk @packgron @tomaszklak
```

Comment:
Why not give everyone from `libtelio-team` same access rights to whole repo?

Reply:
Because this one directory is important from security perspective since changes to workflow files can compromise security of the repo. But it's up to @packgron and @Jauler  to decide if we want this line or not

Reply:
As this is indeed a critical place, for now let's keep it with a list of people who know implications. We can always expand moving forward.